contexts/credentials/v2

```diff
@@ -49,11 +49,6 @@
           "@id": "https://schema.org/description",
           "@type": "http://www.w3.org/2001/XMLSchema#string"
         },
-        "proof": {
-          "@id": "https://w3id.org/security#proof",
```

Comment:
Notice this is a w3id.org IRI... and that proof is not defined here: 

https://www.w3.org/2018/credentials/#proof

Reply:
It seems worth noting that that w3id URI, <https://w3id.org/security#proof>, currently redirects to <https://w3c-ccg.github.io/security-vocab/#proof>, and seems likely to have been intended to (eventually?) redirect to <https://w3.org/2018/security#proof>, based the human-readable content at <https://www.w3.org/2018/credentials/#introduction>.